gui/mainFrame.py

```diff
@@ -38,6 +38,7 @@
 import gui.chromeTabs
 import gui.utils.animUtils as animUtils
 import gui.globalEvents as GE
+import eos.db.saveddata.loadDefaultDatabaseValues as loadDefaultDatabaseValues
```

Comment:
For this you should be able to to from `eos.db.saveddata.loadDefaultDatabaseValues import DefaultDatabaseValues`